### PR TITLE
Set up presets

### PR DESCRIPTION
**Pull request by Henry-Pulver**
This PR also includes a minor fix to visualizing chatbot replies when newlines are in commands

### CODE REVIEW

File: pages/api/swagger-to-actions.ts [hunk 1]
```diff
@@ -156,26 +156,24 @@ export default async function handler(
...
+      "name",
+      actionInserts.map((action) => action.name)
     );
-    const actionInsertResp = await supabase
-      .from("actions")
-      .insert(actionInserts);
-    if (actionInsertResp.error) {
-      throw actionInsertResp.error;
-    }
+  if (actionResp.error) throw actionResp.error;
```
Comment (neuromantic99) [inline pages/api/swagger-to-actions.ts:169]: Not necessary now, but is there something smart we can do with the presets (similar to previous dumb comment)? For example having the presets already in the database then copying them to a new org when a new user wants to use them, might that be faster?

File: public/presets/1/user_description.json [hunk 2]
```diff
@@ -0,0 +1 @@
+""
```
Comment (neuromantic99) [inline public/presets/1/user_description.json:1]: this file meant to be here?

**Review by neuromantic99 — APPROVED** [review]

File: public/presets/1/user_description.json (hunk 2 above)
Comment (Henry-Pulver) [reply public/presets/1/user_description.json:1]: Right now, if it doesn't exist it'll error. I'll stop it from erroring and delete the file :smile:

File: pages/api/swagger-to-actions.ts (hunk 1 above)
Comment (Henry-Pulver) [reply pages/api/swagger-to-actions.ts:169]: Hmm nothing easy I can think of! Maybe we could do something like:
1. Always have 1 org for each preset set up but with no users
2. When a new user joins and chooses a preset, we just associate him with the correct preset org
3. We generate a new preset org not associated with anyone yet (to fill the gap so we get back to state 1)

However, what happens if 2 users join in swift succession and there isn't a spare preset org lying around? Or what happens if there's an error when making the new preset? Etc etc - think this opens up a lot of scope for painful bugs

**Pull request merged**